packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -26,7 +26,7 @@ Okta provides two embedded identity solutions:
    use cases.
 * **Embedded Widget + SDK**: A quick and easy to set up solution that moves most of the the heavy lifting to Okta. Although the amount of code that you need to write is small, many of the most advanced identity use cases (for example, social sign-in, multifactor authentication) are supported out of the box.
 
-<div class="common-image-format">
+<div class="three-quarter border">
```

Comment:
```suggestion
<div class="three-quarter">
```

Reply:
Agreed, that looks better. Done in next commit